### PR TITLE
Cross link also from Typed doc pages to Classic

### PR DESCRIPTION
**Pull request by helena**
#24717
* To quickly route everyone to the right place
* because most people land in the docs from a Google search or link from a blog post

### CODE REVIEW

File: akka-docs/src/main/paradox/typed/cluster-singleton.md [hunk 1]
```diff
@@ -1,5 +1,11 @@
 # Cluster Singleton
 
+@@@ note
+
+For the Akka Classic API see @ref:[Classic Cluster Singleton](../cluster-singleton.md)
+
+@@@
```
Comment (johanandren) [inline akka-docs/src/main/paradox/typed/cluster-singleton.md]: I think we said we don't want the classic cross-link up top/most prominent.

Comment (johanandren) [reply akka-docs/src/main/paradox/typed/cluster-singleton.md]: I take it back!

**Review by johanandren — APPROVED** [review]

**Review by chbatey — APPROVED** [review]

**Comment by helena** [comment]
Updated

**Review by patriknw — APPROVED** [review]
LGTM

**Review by johanandren — APPROVED** [review]

**Pull request merged**